source/connect/authentication-connection.txt

```diff
@@ -13,3 +13,129 @@ Authentication Connection Tab
    :class: singlecol
 
 
+The :guilabel:`Authentication` tab allows you to connect deployments that 
+require authentication. To learn about authentication mechanisms within MongoDB,
+see :driver:`Authentication Mechanisms </go/current/fundamentals/auth/>`
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Authentication` tab.
+
+      Select your authentication method from the following options: 
+      
+      - :ref:`Username/Password <username-password>`
+      
+      - :ref:`X.509 <x509>`
+      
+      - :ref:`Kerberos <kerberos-authentication>`
+      
+      - :ref:`LDAP <ldap-authentication>`
+      
+      - :ref:`AWS IAM <aws-iam-authentication>`
+
+      .. _username-password:
+
+      Username/Password 
+      ~~~~~~~~~~~~~~~~~
+
+      Provide the following information:
+
+      - Username
+
+      - Password
+
+      - (Optional) :manual:`Authentication Database </core/security-users/#authentication-database>`
+
+      - Select an Authentication Mechancism:
+         
+        - Default
+
+          MongoDB uses :manual:`SCRAM </core/security-scram/>` as the default 
+          authentication mechanism.
+
+        - :manual:`SCRAM-SHA-1 </core/security-scram/>`
+
+        - :manual:`SCRAM-SHA-256 </core/security-scram/>`
+
+      .. _x509:
+
+      X.509
+      ~~~~~
+
+      Select :guilabel:`X.509` if the deployment uses :manual:`X.509 
+      </core/security-x.509/>` as its authentication mechanism.
+
+      .. _kerberos-authentication:
+
+      Kerberos
+      ~~~~~~~~
+
+      Select :guilabel:`Kerberos` if the deployment uses :manual:`Kerberos 
+      </core/kerberos/>` as its authentication mechanism. 
+
+      Provide the following information:
+
+      - :manual:`Principal </core/kerberos/#principals>`
+
+      - Service Name
+
+      - (Optional) Canonicalize Host Name
+
+        When you enable this setting, Kerberos uses the canonicalized form of 
+        the host name ``(cname)`` when constructing the principal for |compass|
+
+        For more information on principal name canonicalization in Kerberos, see 
+        this `RFC document <https://tools.ietf.org/html/rfc6806.html>`__.
+
+      - Service Realm *waiting on tech clarification*
```

Comment:
It looks like we're missing the "Provide Password Directly" option?

Reply:
Done.